public/app/features/search/components/DashboardSearch.tsx

```diff
@@ -45,7 +51,7 @@ function DashboardSearchNew({ onCloseSearch }: Props) {
           <div>
             <input
               type="text"
-              placeholder="Search dashboards by name"
+              placeholder={includePanels ? 'Search dashboards and panels by name' : 'Search dashboards by name'}
```

Comment:
if the placeholder text did not indicate state, this could be held entirely in `<SearchView .../>`